packages/frontend/src/components/NavBar/GlobalSearch/index.tsx

```diff
@@ -132,6 +133,8 @@ const GlobalSearch: FC<GlobalSearchProps> = ({ projectUuid }) => {
     const { track } = useTracking();
     const project = useProject(projectUuid);
 
+    const canUserManageValidation = useValidationUserAbility(projectUuid);
```

Comment:
I like that! its just not clear from the name `useValidationUserAbility` that its about `Manage` permission

Reply:
hmmm I see! I would like to keep it because it matches the current usage of `user.ability`